src/viur/toolkit/importer/importer.py

```diff
@@ -521,15 +521,15 @@ def set_value(val: t.Any, lang: str | None) -> None:
                     elif bone_value[lang] != val:
                         changes += 1
 
-                    if bone.multiple:
+                    if not bone.multiple:
                         if val is None:
                             val = []
-                        elif isinstance(val, (str, numbers.Number)):
+                        elif isinstance(val, (str, bool, numbers.Number)):
```

Comment:
@phorward FYI: `bool` is an subclass of `int` which was already covered by `numers.Number`. But okay for me, sometimes explicit is better than implicit.

Reply:
ok you can change it back if you want. maybe a comment then?

Reply:
NO, we can keep it as you changes.